### ISSUE TITLE
F.log_softmax raises nan values

### ISSUE THREAD

**Issue by wkentaro** [pytorch/pytorch#795]
I'm working on image classification task, and get nan in the loss after the cross_entropy.
By analyzing with ipdb module, I got the `F.log_softmax` raises nan values:

```
ipdb> np.isnan(input.data.cpu().numpy()).sum()
0
ipdb> np.isnan((F.log_softmax(input).data.cpu().numpy())).sum()
7733
ipdb> np.isnan((F.softmax(input).data.cpu().numpy())).sum()
0
ipdb> np.isnan(np.log(F.softmax(input).data.cpu().numpy())).sum()
0
```

Is this one of the known issues like https://github.com/pytorch/pytorch/issues/773?
Or am I missing something?

**Comment by apaszke** [comment]
What's the size of the input? Can you print the max and min value?

**Comment by wkentaro** [comment]
Is below enough information?
```
ipdb> input.size()
torch.Size([1, 21, 333, 500])
ipdb> type(input.data)
<class 'torch.cuda.FloatTensor'>
ipdb> input.data.min()
-40.114463806152344
ipdb> input.data.max()
94.69690704345703
```

**Comment by apaszke** [comment]
It should have been fixed by #798.

**Comment by wkentaro** [comment]
Thanks. It seems fixed when I install pytorch from source.
